src/x11.cc

```diff
@@ -449,6 +453,40 @@ static void update_workarea(void)
 	workarea[1] = 0;
 	workarea[2] = display_width;
 	workarea[3] = display_height;
+
+#ifdef BUILD_XINERAMA
+	/* if xinerama is being used, adjust workarea to the head's area */
+	int useless1, useless2;
+	if (!XineramaQueryExtension(display, &useless1, &useless2)) {
+		return; /* doesn't even have xinerama */
+	}
+
+	if (!XineramaIsActive(display)) {
+		return; /* has xinerama but isn't using it */
+	}
+
+	int heads = 0;
+	XineramaScreenInfo *si = XineramaQueryScreens(display, &heads);
+	if (!si) {
+		puts("warning: XineramaQueryScreen returned NULL, ignoring head settings");
```

Comment:
Please use the macro `NORM_ERR()` here (from https://github.com/brndnmtthws/conky/blob/master/src/logging.h#L73-L77).